package.json

```diff
@@ -11,8 +11,8 @@
     "types": "tsc --pretty",
     "test": "cross-env ENVFILE=config/.env.local jest --color",
     "start:android": "cross-env ENVFILE=config/.env.local react-native run-android",
-    "start:ios": "cross-env ENVFILE=config/.env.local react-native run-ios --simulator='iPhone 12'",
-    "start:ios:ipad": "cross-env ENVFILE=config/.env.local react-native run-ios --simulator='iPad (9th generation)'",
+    "start:ios": "cross-env ENVFILE=config/.env.local react-native run-ios --simulator='iPhone 14'",
```

Comment:
This will mean that our setup guide needs to reference Xcode 14 onwards as a requirement, fyi